src/scripts/main.js

```diff
@@ -358,3 +359,31 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+function calculateAge(born, died) {
+  return died - born;
+}
+
+function calculateCentury(year) {
+  return Math.ceil(year / 100);
+}
+
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that there is an element with the class 'dashboard' in your HTML. Otherwise, 'table' will be null, and the code will throw an error when trying to append rows.